flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go

```diff
@@ -69,6 +70,14 @@ func GetExecutionEnvVars(id pluginsCore.TaskExecutionID) []v1.EnvVar {
 		// },
 	}
 
+	if consoleURL != "" {
+		envVars = append(envVars, v1.EnvVar{
+			Name: "FLYTE_EXECUTION_URL",
+			// TODO: should we use net/url to build this url?
+			Value: fmt.Sprintf("%s/projects/%s/domains/%s/executions/%s/nodeId/%s/nodes", consoleURL, nodeExecutionID.Project, nodeExecutionID.Domain, nodeExecutionID.Name, id.GetUniqueNodeID()),
```

Comment:
Maybe trim suffix `/` from consoleURL before appending another `/`

Reply:
good call. Added a couple of tests to cover this.